docs/update.js

```diff
@@ -30,27 +29,68 @@ const damageWeightValueId = 'damage weight value';
  * @param {google.map.Maps} map
  */
 function update(map) {
-  for (const toggle of toggles.keys()) {
-    const newValue = Number(getValue(toggle));
-    if (hasErrors(newValue, toggle)) {
-      return;
-    } else {
-      toggles.set(toggle, newValue);
-      setValue(toggle, newValue);
-    }
+  getUpdatedValue(povertyThresholdKey);
+  if (hasDamageAsset) {
+    getUpdatedValue(damageThresholdKey);
+    getUpdatedValue(povertyWeightKey);
   }
 
   removeScoreLayer();
-  createAndDisplayJoinedData(
-      map, toggles.get('poverty threshold'), toggles.get('damage threshold'),
-      toggles.get('poverty weight'));
+  createAndDisplayJoinedData(map, Promise.resolve(getValuesAsArray()));
   // clear old listeners
   google.maps.event.clearListeners(map, 'click');
   google.maps.event.clearListeners(map.data, 'click');
 }
 
 /**
- * Creates the form for toggling the equation.
+ * Pulls value from input box and
+ * @param {string} toggle
+ */
+function getUpdatedValue(toggle) {
+  const newValue = Number(getValue(toggle));
+  if (!hasErrors(newValue, toggle)) {
+    toggles.set(toggle, newValue);
+  }
+}
+
+// Set in setUpInitialToggleValues.
+let hasDamageAsset = null;
+
+/**
+ * Initializes damage-related toggle values based on whether or not we have
+ * a damage asset.
+ * @param {Promise<Object>} disasterMetadataPromise
+ * @param {google.map.Maps} map
+ * @return {Promise<Array<number>>} returns all the toggle initial values.
+ */
+function setUpToggles(disasterMetadataPromise, map) {
+  const togglesSetPromise = disasterMetadataPromise.then((doc) => {
+    hasDamageAsset = doc.data()['asset_data']['damage_asset_path'];
+    if (hasDamageAsset) {
+      toggles.set(damageThresholdKey, 0.5);
+      toggles.set(povertyWeightKey, 0.5);
+    }
+    return getValuesAsArray();
+  });
+  togglesSetPromise.then(() => createToggles(map));
```

Comment:
any reason not to do this in then block above? and then inline toggleSetPromise?

Reply:
I didn't want other stuff chained off togglesSetPromise to have to wait on createToggles to finish, even if that's quick.

Reply:
As discussed offline, this is UI work, so should probably be prioritized, since the Promise consumer presumably isn't kicking off more async work based on this value.

Reply:
Done.